### PR TITLE
Soap and Rest

### PR DESCRIPTION
**Pull request by Yanetta**

### CODE REVIEW

**Comment by Artsapollo** [comment]
Maybe I'm not good in REST and SOAP so far, but it looks good

File: src/main/java/workWithRest/MateGroupService.java [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
-import xmljson.Person;
+        import xmljson.Person;
 
-import javax.ws.rs.core.Response;
+        import javax.ws.rs.core.Response;
 
 public interface MateGroupService {
     public Response getMateGroup();
-    public Response addStudents(int groupId, Person person);
```
Comment (SerhiiPasko) [inline src/main/java/workWithRest/MateGroupService.java:10]: Task was add new method, don't change existing

File: src/main/java/workWithRest/MateGroupServiceClient.java [hunk 2]
```diff
@@ -53,21 +50,22 @@ private static void showInfo(HttpURLConnection conn) throws IOException {
...
-        private static HttpURLConnection addStudent(String pathParam)
+        private static HttpURLConnection changeTeacher(String pathParam)
                 throws MalformedURLException, IOException, ProtocolException {
             URL url = new URL("http://localhost:9998/rs/mate/" + pathParam);
             HttpURLConnection conn = (HttpURLConnection) url.openConnection();
             conn.setDoOutput(true);
             conn.setRequestMethod("PUT");
             conn.setRequestProperty("Content-Type", "application/json");
-            String input = "{\"name\": \"Iza\",\"yearOfBorn\": 1997	}";
+            String input = "Sasha";
```
Comment (SerhiiPasko) [inline src/main/java/workWithRest/MateGroupServiceClient.java:60]: It real work? You must put json. This is RAW text. You will be show me that at lesson

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comments. Also I don't see soap screen

**Review by Artsapollo — APPROVED** [review]

**Pull request merged**